lib/ibanity/api/customer_access_token.rb

```diff
@@ -1,8 +1,10 @@
 module Ibanity
   class CustomerAccessToken < ::Ibanity::BaseResource
-    def self.create(idempotency_key: nil, **attributes)
+    def self.create(application_customer_reference, idempotency_key: nil, **attributes)
```

Comment:
while it would improve the API, we should pass it as a keyword param to stay consistent